plugins/org.yakindu.sct.doc.user/src/user-guide/quick_ref.textile

```diff
@@ -0,0 +1,231 @@
+| !images/docu_editor_palette_300_tool_exit_point.png(Editor palette element tool "Exit point")! | Exit points are used to leave a composite state and are the counterparts of entry points. Read more about them in section "Composite states":#quick_ref_composite_states. |
+| !images/docu_editor_palette_310_tool_choice.png(Editor palette element tool "Choice")! | A choice node is used to model a conditional path. Read more in section "Choices":#quick_ref_choices. |
+| !images/docu_editor_palette_320_tool_synchronization.png(Editor palette element tool "Synchronization")! | Synchronization nodes are used to model forks and joins in combination with orthogonal states. Read more in section "Orthogonality":#quick_ref_orthogonality. |
+
+
+h2(#quick_ref_definition_section). Statechart definition section
+
+The _definition section_ of the statechart defines which entities of the statechart, like variables, events and operations, are accessible from the outside and which ones are only used internally. For this, a definition section can declare an internal scope and multiple interfaces. 
+
+!(tiny)images/quick_ref_def_section.png(Overview of a statechart's definition section)!
+
+h3(#quick_ref_interface_scope). Statechart interfaces
+
+A statechart interface declares the entities that are externally visible. These are the elements by which the client code can interact with the statechart. A statechart can declare multiple interfaces with different names. The unnamed interface is also called the default interface.
+
+|{width:420px}_. Interface Element Declaration |_. Meaning |
+| <code>in event SwitchOn</code> | Incoming event, supposed to be raised by the client code and processed by the state machine to evaluate potential state transitions. |
+| <code>in event Slider : integer</code> | Incoming event with payload of type @integer@ |
+| <code>out event Finish</code> | Outgoing event, supposed to be raised by the state machine and delivered to the outside. |
+| <code>out event Error : string</code> | Outgoing event with payload of type @string@. Can be raised by a transition or state reaction with @raise Error : "Some error message"@ |
+| <code>var brightness : integer</code> | Variable, used to store some data. Can be changed by the state machine and by the client code. |
+| <code>var brightness : integer = 3</code> | Variable with initial value. |
+| <code>var readonly brightness : integer</code> | Variable marked as readonly to ensure it is not changed by the client code. |
+| <code>const PI : real = 3.14</code> | Constant, used to store some immutable data that is not changeable by the client code or the state machine. Constants must have an initial value. |
+| <code>operation average(a : real, b : real) : real</code> | Operation, connects a state machine to the outside world by making external behaviour accessible. Implementation needs to be provided by the client code. |
```

Comment:
`| <code>operation doStuff()</code> | Operation, connects a state machine to the outside world by making external behaviour accessible. Implementation needs to be provided by the client code. |`

`| <code>operation average(a : real, b : real) : real</code> | Operation with two parameters a and b of type real and return type real. |`